stripe/src/test/java/com/stripe/android/view/CountryTextInputLayoutTest.kt

```diff
@@ -136,6 +138,41 @@ class CountryTextInputLayoutTest {
         )
     }
 
+    @Test
+    fun `when screen rotates then selected country should carry over`() {
+        val application = ApplicationProvider.getApplicationContext<Application>()
+        application.setTheme(R.style.StripePaymentSheetDefaultTheme)
+        val oldCountryTextInputLayout = CountryTextInputLayout(
+            application
+        )
```

Comment:
FYI if you want to create an Activity scoped View, you can do this:

```
        activityScenarioFactory.createView { activity ->
            CountryTextInputLayout(activity)
        }
```

Reply:
thanks for the tip! updated